docs/node/activities.md

```diff
@@ -0,0 +1,47 @@
+# Activities
+
+In Temporal, Activities are typically used to represent side effects, like making an HTTP request.
+Unlike [Workflows](/docs/node/logging#logging-from-workflows-and-activities), activities run in the standard Node.js environment, not an [isolate](https://www.npmjs.com/package/isolated-vm).
+So any code that needs to talk to the outside world, including importing modules, needs to be in an Activity.
+
+### Overview
+
+Below is a simple Activity that accepts a string parameter, appends a word to it, and returns the result.
+The Temporal Node SDK looks for any `.js` files in the `src/activities` directory, and automatically registers any exported functions as Activities.
```

Comment:
Not accurate, in the package initializer, it's `.ts` in `src/activites` and the compiled code in `lib/activities` (configurable).

Reply:
Worth it to always use `lib/activities` rather than `src/activities`? If you're not using package initializer, either seems to work because of what [`activitiesPath` resolves to](https://github.com/temporalio/sdk-node/blob/9176b27270f7da77972ceb18c9c624f12e11ee93/packages/worker/src/worker.ts#L329)